js/body_mixin.ts

```diff
@@ -0,0 +1,230 @@
+/** @module fly
+ */
+import { parse as queryParse } from "query-string";
+import {
```

Comment:
in order to build the library properly, I believe this needs to be a namespaced import:

```ts
import * as domTypes from "./dom_types";
```